Example/FirestoreSample/.gitignore

```diff
@@ -0,0 +1,66 @@
+# Logs
```

Comment:
We've avoided adding lower level `.gitignore` files to the repo. Is it possible to add just what's needed to the top-level one?